php/Exchange.php

```diff
@@ -104,6 +104,7 @@ class Exchange {
         'bw',
         'bybit',
         'bytetrade',
+        'bytex',
```

Comment:
@kroitor Is this automatically put in there by the Travis Bot? So should it be removed from this PR?

Reply:
@samgermain yes, it will be done automatically during the build, so we don't have to add it manually and we can cleanup that file.